### ISSUE TITLE
Build failure: qutebrowser

### ISSUE THREAD

**Issue by KiaraGrouwstra** [NixOS/nixpkgs#262807]
### Steps To Reproduce

1. build `qutebrowser`

### Build log

```
error:
       … while calling the 'derivationStrict' builtin

         at /builtin/derivation.nix:9:12: (source not available)

       … while evaluating derivation 'home-manager-generation'
         whose name attribute is located at /nix/store/9avrpkh90hh5nciins6xz27xnf5rra49-source/pkgs/stdenv/generic/make-derivation.nix:300:7

       … while evaluating attribute 'buildCommand' of derivation 'home-manager-generation'

         at /nix/store/9avrpkh90hh5nciins6xz27xnf5rra49-source/pkgs/build-support/trivial-builders/default.nix:87:14:

           86|       enableParallelBuilding = true;
           87|       inherit buildCommand name;
             |              ^
           88|       passAsFile = [ "buildCommand" ]

       (stack trace truncated; use '--show-trace' to show the full trace)

       error: stem-1.8.2 not supported for interpreter python3.11
```

### Additional context

this issue appears to match a recent [comment added to the `stem` nix package](https://github.com/NixOS/nixpkgs/commit/f27432537421d6c67977a4e50e16269df5a14484)

### Notify maintainers

@rhendric (for lack of listed maintainers)

### Metadata

Please run `nix-shell -p nix-info --run "nix-info -m"` and paste the result.

```console
[user@system:~]$ nix-shell -p nix-info --run "nix-info -m"
 - system: `"x86_64-linux"`
 - host os: `Linux 6.1.59, NixOS, 23.11 (Tapir), 23.11.20231022.e7ce711`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.17.1`
 - channels(kiara): `"home-manager"`
 - channels(root): `"home-manager, nixos-23.05"`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`
```

**Comment by rhendric** [comment]
qutebrowser's maintainers: @jagajaga @rnhmjoj @ebzzry @dotlambda @nrdxp

(Unsubscribing as I don't have anything to do with this software; re-@ me if you want my attention.)

**Comment by nrdxp** [comment]
from a brief examination it looks like the stem library is not available for python 3.10. No experience with this package but it looks like it is for a userscript. I see @dotlambda has already opened a PR to fix it, but according to the libraries own website, it is no longer maintained, so perhaps we should consider just removing it?
